main/src/modules/contextualLayer/goalManager/include/goalManager.h

```diff
@@ -0,0 +1,56 @@
+#include "wrdac/clients/icubClient.h"
+
+using namespace std;
+using namespace yarp::os;
+using namespace wysiwyd::wrdac;
+
+class GoalManager : public RFModule {
+private:
+
+    ICubClient *iCub;
+    double      period;
+    Port        rpc;
+    Port        portToBehavior;
+    BufferedPort<Bottle>        port_behavior_context;
+    BufferedPort<Bottle>        goal_need_port;
+    Bottle goal_list;
+    int id;
+    int fulfill;
+    Bottle* current_goal;
+    string allostasis_input_port;
+
+    bool bShouldListen;
+
+    std::string      grammarToString(std::string sPath);
+    std::string      MainGrammar;
```

Comment:
This method + variable are not used. Please delete (`bShouldListen`, `grammarToString`, `MainGrammar`)